docs/api-reference.md

```diff
@@ -14,7 +14,7 @@ class EnumType(SchemaBindable):
     ...
 ```
 
-[Bindable](bindables.md) mapping Python values to enumeration members in a [GraphQL schema](https://graphql-core-3.readthedocs.io/en/latest/modules/type.html#graphql.type.GraphQLSchema).
+[Bindable](bindables.md) mapping Python values to enumeration members in a [GraphQL schema](https://graphql-core-3.readthedocs.io/en/latest/modules/type.html#graphql.type.[GraphQLSchema](https://graphql-core-3.readthedocs.io/en/latest/modules/type.html#graphql.type.GraphQLSchema)).
```

Comment:
Reference to url is "duplicated", looks like `GraphQLSchema` in url, from existing doc, was treated as another token. The same for other occurrences.

Reply:
Great catch. I've reordered replaces in doc generator, but I'll keep it on back of my head that better solution may eventually be needed.